app/views/common_parameters/_puppetclass_parameter.html.erb

```diff
@@ -16,13 +16,13 @@
                         :placeholder => _("Value") %>
           <%= text_field_tag '', (f.object.lookup_key.key_type rescue 'unknown'), :class=> 'hide',  :'data-property' => 'type', :disabled => true, :title => _('Parameter type') %>
         </td>
-        <td class="col-md-2">
+        <td class="col-md-5">
           <%= f.check_box :use_puppet_default, :'data-property' => 'use_puppet_default',
                         :disabled => (not authorized_via_my_scope("host_editing", "edit_params")),
                         :onchange=>'toggleUsePuppetDefaultValue(this, "value")' %>
-          <%= popover('', _('Do not send this parameter via the ENC. Puppet will use the value defined in the puppet manifest for this parameter')) %>
+          <%= popover('', _("Do not send this parameter via the ENC.<br> Puppet will use the value defined in the puppet manifest for this parameter")) %>
```

Comment:
This also appears in app/views/lookup_keys/_value.html.erb:7 and app/views/lookup_keys/_fields.html.erb:18. However only sometimes is it a popover - consider extracting to helper or otherwise changing to ensure consistency.